ci/upgrade/cstor-operator.tmp.yaml

```diff
@@ -41,7 +41,7 @@ spec:
       containers:
       - name: cspc-operator
         imagePullPolicy: IfNotPresent
-        image: imageorg/cspc-operator-amd64:testimage
+        image: imageorg/cspc-operator:testimage
```

Comment:
What does the `imageorg` mean here?

Reply:
This is a template and the imageorg lets you test on different docker repos like openebs/ or shubham/. It defaults to openebs